### DISCUSSION TITLE
Include target as result?

### DISCUSSION THREAD

**Discussion by rieje** [sharkdp/fd#1557] [Q&A]
Quick question--is it possible to include target as result? I have:

    fd -t e -t d . "$src" -X rmdir '{}'

but I also want `$src` to be removed in one `rmdir` call.

I tried 

    fd -t e -t d . "$src" -X rmdir {} "$src"

but this only removes `$src` works if there are results from `fd`. 

I'm implementing an rsync version of `mv`.

**Comment by tavianator** [comment]
See #364 for previous discussion about this
